.github/workflows/bumpVersions.yml

```diff
@@ -0,0 +1,75 @@
+---
+name: Bump component versions
+
+on:
+  workflow_dispatch:
+    inputs:
+      jira_id:
+        description: ticket reference for the requested update (OPSEXP-1234)
+        required: true
+  push:
+
+permissions:
+  contents: "write"
+  pull-requests: "write"
+
+jobs:
+  updatecli:
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 1
+      matrix:
+        version:
+          - latest
+          - 7.3.N
+          - 7.2.N
+          - 7.1.N
+          - 7.0.N
+          #- community
+    env:
+      MATRIX_VALUES_OUTPUT: /tmp/${{ matrix.version }}-values.yaml
+      UPDATECLI_MANIFEST_FILE: ./updatecli.d/enterprise-manifest.tpl
+      JIRA_ID: ${{ inputs.jira_id }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Updatecli
+        uses: updatecli/updatecli-action@v2
+
+      - name: Prepare values file for ${{ matrix.version }}
+        working-directory: ./updatecli.d/
+        run: |
+          yq -e e 'explode(.) | .matrix["${{ matrix.version }}"]' supported-matrix.yaml \
+          | tee > "$MATRIX_VALUES_OUTPUT"
+
+      - name: Get today date for cache key
+        id: get-date
+        run: |
+          echo "date=$(/bin/date -u "+%Y%m%d")" >> $GITHUB_OUTPUT
+        shell: bash
+
+      - name: Cache updatecli
+        id: updatecli-cache
+        uses: actions/cache@v3
+        with:
+          path: /tmp/updatecli
+          key: updatecli-${{ steps.get-date.outputs.date }}
+
+      - name: Run Updatecli in dry run mode
+        env:
+          QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+          QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+          UPDATECLI_GITHUB_TOKEN: ${{ secrets.BOT_GITHUB_TOKEN }}
+        run: >-
+          JIRA_ID=${JIRA_ID:-DRAFT}
+          updatecli diff -c "$UPDATECLI_MANIFEST_FILE" -v "$MATRIX_VALUES_OUTPUT"
+
+      - name: Run Updatecli in apply mode
+        env:
+          QUAY_USERNAME: ${{ secrets.QUAY_USERNAME }}
+          QUAY_PASSWORD: ${{ secrets.QUAY_PASSWORD }}
+          UPDATECLI_GITHUB_TOKEN: ${{ secrets.BOT_GITHUB_TOKEN }}
+        run: >-
+          JIRA_ID=${JIRA_ID:-DRAFT}
+          updatecli apply -c "$UPDATECLI_MANIFEST_FILE" -v "$MATRIX_VALUES_OUTPUT"
```

Comment:
I didn't like `manifest show` because it prints credentials (that ofc get obfuscated in the logs but better be safe than sorry)

at this point I am not sure I am keeping the diff step since I've seen that in case of failures I get the same errors with both commands (in the end diff is more helpful when testing locally to avoid opening a PR every time)